install/preview/entrypoint.sh

```diff
@@ -136,6 +136,18 @@ yq eval-all -i 'del(.spec.template.spec.initContainers[0])' /var/lib/rancher/k3s
 for f in /var/lib/rancher/k3s/server/manifests/gitpod/*.yaml; do (cat "$f"; echo) >> /var/lib/rancher/k3s/server/manifests/gitpod.yaml; done
 rm -rf /var/lib/rancher/k3s/server/manifests/gitpod
 
+# waits for gitpod pods to be ready, and manually runs the `gitpod-telemetry` cronjob
+run_telemetry(){
+  # wait for the k3s cluster to be ready and Gitpod workloads are added
+  sleep 100
```

Comment:
Magic sleep time feels always like a hack. What if it takes longer than 100 for some reason? However, maybe good enough for this use case.

Reply:
Yep, It is a hack but Just for atleast one Gitpod pod to be just added seems like a valid wait time. Other hacks involve looping on `kubectl wait` command https://github.com/kubernetes/kubernetes/issues/83242#issuecomment-1029256777 🤔

Reply:
Thinking more about this. A lot of these kind of hacks, seem to be cause of the fact that we reun `k3s` with manifests added. How about we run the cluster parallely, and apply manifests on our own. We can then also remove `prettylog` and instead print statements as they happen, and we'll have much more easy time handling the script 🤔

Reply:
Makes sense. :+1: